command/command.go

```diff
@@ -55,6 +75,53 @@ func (c *Command) processCommand() error {
 	}
 
 	var err error
+	
+	if c.OsType != types.Ubuntu && c.OsType != types.Flatpak && c.Command != "init" && c.ContainerName == "" {
+		cmd := exec.Command("/bin/bash", "-c", "apx subsystems list | grep ^" + c.ContainerSubsystem() + "$")
+    		cmd.Stdin = os.Stdin
+    		cmd.Stderr = os.Stderr
+    		if err != nil {
+       			fmt.Println("Error 1")
+    		}
+    		if err := cmd.Start(); err != nil {
+        		fmt.Println("Error 2")
+    		}
+    		if err != nil {
+        		fmt.Println("Error 3")
+    		}
+    		if err := cmd.Wait(); err != nil {
+    			fmt.Println("Warning: Subsystem hasn't been pre-initialized, initializing...")
+        		cmd_exec := exec.Command("/bin/bash", "-c", "apx subsystems new -n " + c.ContainerSubsystem() + " -s " + c.ApxSubsystem())
+        		cmd_exec.Stdout = os.Stdout
+			if err := cmd_exec.Run(); err != nil {
+  				fmt.Println("Apx Error: ", err)
+			}
+    		}
+	}
+	
+	if c.OsType != types.Ubuntu && c.OsType != types.Flatpak && c.Command != "init" && c.ContainerName != "" {
+		cmd := exec.Command("/bin/bash", "-c", "apx subsystems list | grep ^" + c.ContainerName + "$")
+    		cmd.Stdin = os.Stdin
+    		cmd.Stderr = os.Stderr
+    		if err != nil {
+       			fmt.Println("Error 1")
+    		}
+    		if err := cmd.Start(); err != nil {
+        		fmt.Println("Error 2")
+    		}
+    		if err != nil {
+        		fmt.Println("Error 3")
+    		}
+    		if err := cmd.Wait(); err != nil {
+    			fmt.Println("Warning: Subsystem hasn't been pre-initialized, initializing...")
```

Comment:
We shouldn't be using an error to do this logic, instead parse the output of the command and react accordingly. Other actual errors could also happen e.g broken podman where we don't want to then call the subsystem new command.